magento/adobe-stock-integration
# User quota is not updated during authorization

### Description (*)


### Fixed Issues (if relevant)

1. magento/adobe-stock-integration#434: User quota is not updated during authorization
2. ...

### Manual testing scenarios (*)
1. Login to admin panel
2. Open Magento Media Gallery (i.e. go to Cms -> Pages, edit the page and insert image)
3. Click "Search Adobe Stock" button to open images grid
4. Click on "Sign in" link in top right corner and login with the Adobe Stock account with credits
5. Click on the logged-in user name (that appears instead of "Sign in" link) to show user profile popup

### Expected result (*)
Credits/Images are showing the default value "0", they are updated only after the page is refreshed
